src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -145,211 +94,17 @@ export default function ConfirmedAppointmentDetailsPage() {
     );
   }
 
-  if (
-    appointment.vaos.isCOVIDVaccine ||
-    appointment.vaos.isPhoneAppointment ||
-    !appointment.vaos.isVideo
-  ) {
-    return (
-      <>
-        <DetailsVA appointment={appointment} facilityData={facilityData} />
-        <CancelAppointmentModal
-          {...cancelInfo}
-          onConfirm={() => dispatch(confirmCancelAppointment())}
-          onClose={() => dispatch(closeCancelAppointment())}
-        />
-      </>
-    );
-  }
-
-  const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
   const isVideo = appointment.vaos.isVideo;
-  const isPastAppointment = appointment.vaos.isPastAppointment;
-  const facility = facilityData?.[locationId];
-  const isInPersonVAAppointment = !isVideo;
-  const isCovid = appointment.vaos.isCOVIDVaccine;
-  const canceler = appointment.description?.includes('CANCELLED BY PATIENT')
-    ? 'You'
-    : facility?.name || 'Facility';
-
-  const header = formatHeader(appointment);
-
-  const showInstructions =
-    isInPersonVAAppointment &&
-    PURPOSE_TEXT.some(purpose =>
-      appointment?.comment?.startsWith(purpose.short),
-    );
-
-  const calendarData = getCalendarData({
-    appointment,
-    facility: facilityData[locationId],
-  });
-
-  let infoAlert = null;
-  if (canceled) {
-    infoAlert = (
-      <InfoAlert status="error" backgroundOnly>
-        {`${canceler} canceled this appointment.`}
-      </InfoAlert>
-    );
-  } else if (isPastAppointment) {
-    infoAlert = (
-      <InfoAlert status="warning" backgroundOnly>
-        This appointment occurred in the past.
-      </InfoAlert>
-    );
-  } else if (showConfirmMsg) {
-    infoAlert = (
-      <InfoAlert backgroundOnly status="success">
-        <strong>Your appointment has been scheduled and is confirmed.</strong>
-        <br />
-        <div className="vads-u-margin-y--1">
-          <Link
-            to="/"
-            onClick={() => {
-              recordEvent({
-                event: `${GA_PREFIX}-view-your-appointments-button-clicked`,
-              });
-            }}
-          >
-            View your appointments
-          </Link>
-        </div>
-        <div>
-          <Link to="/new-appointment">New appointment</Link>
-        </div>
-      </InfoAlert>
-    );
-  }
+  const isCommunityCare = appointment.vaos.isCommunityCare;
```

Comment:
Eventually I want to put CC details in here, too, so this is just to make it clear what types we expect right now.